src/common/item-types/settings-item-config.ts

```diff
@@ -0,0 +1,34 @@
+//////////////////////////////////////////////////////////////////////////////////////////
+//   _  _ ____ _  _ ___  ____                                                           //
+//   |_/  |__| |\ | |  \ |  |    This file belongs to Kando, the cross-platform         //
+//   | \_ |  | | \| |__/ |__|    pie menu. Read more on github.com/menu/kando           //
+//                                                                                      //
+//////////////////////////////////////////////////////////////////////////////////////////
+
+// SPDX-FileCopyrightText: Simon Schneegans <code@simonschneegans.de>
+// SPDX-License-Identifier: MIT
+
+import i18next from 'i18next';
+
+import { IMenuItem } from '..';
+import { IItemConfig } from '../item-config-registry';
+
+/**
+ * This class provides the configuration widgets for opening the settings items. There is
+ * no configuration for this widget.
+ */
+export class SettingsItemConfig implements IItemConfig {
+  /** @inheritdoc */
+  public getTipOfTheDay(): string {
+    const tips = [i18next.t('items.settings.tip-1')];
+
+    return tips[Math.floor(Math.random() * tips.length)];
+  }
+
+  /** @inheritdoc */
+  public getConfigWidget(item: IMenuItem): DocumentFragment | null {
+    if (item.data == null) {
+      return null;
+    }
```

Comment:
I think the if is not required:
```suggestion
    return null;
```